### DISCUSSION TITLE
filament-media-manager

### DISCUSSION THREAD

**Discussion by bose733** [filamentphp/filament#14699] [Help]
### Package

Panel builder

### Package Version

v3.0.0

### How can we help you?

composer require tomatophp/filament-media-manager
When i install this package there have no Breadcrumbs 
how to fix it??
Documentation : :  https://filamentphp.com/plugins/3x1io-tomato-media-manager

like as this "breadcrumb" missing
![breadcrumb](https://github.com/user-attachments/assets/b32e8f29-cb0b-446e-af4e-f4b6823c6769)

**Comment by leandrocfe** [comment]
Please ask [on discord](https://discord.com/channels/883083792112300104/1265002822605344871)
